deploy/crds/policy.open-cluster-management.io_placementbindings.yaml

```diff
@@ -58,8 +62,15 @@ spec:
                   type: string
                 name:
```

Comment:
Could you please add `minLength` on these properties?

Reply:
done